perf.groovy

```diff
@@ -420,7 +408,7 @@ def static getFullThroughputJobName(def project, def os, def isPR) {
 
             def osGroup = getOSGroup(os)
             def runType = isPR ? 'private' : 'rolling'
-            def benchViewName = isPR ? 'coreclr private \$BenchviewCommitName' : 'coreclr rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
+            def benchViewName = isPR ? 'coreclr-throughput private \$BenchviewCommitName' : 'coreclr-throughput rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
```

Comment:
Perhaps I missed it, but is this always a throughput run?  Also, the comment right above this for the input parameter BenchViewCommitName is no longer correct with this change.